app/models/host/managed.rb

```diff
@@ -427,6 +445,10 @@ def info
     info_hash['parameters'] = param
     info_hash['environment'] = param["foreman_env"] if Setting["enc_environment"] && param["foreman_env"]
 
+    host_facets.each do |facet|
```

Comment:
Maybe this whole `info` method should be part of the Puppet facet, and we can extract it in that PR instead of offering an endpoint in the generic host facet